### PR TITLE
Add OCSP config and PKCS#11 OCSP support

### PR DESCRIPTION
**Pull request by jsha**

### CODE REVIEW

File: crypto/pkcs11key/pkcs11key_stub.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Some Go tools will complain "no buildable Go source files in ..." because
```
Comment (kisom) [inline crypto/pkcs11key/pkcs11key_stub.go]: If we're putting this file in here instead of in #226, the comment from that PR should be ported here.

**Comment by jsha** [comment]
Updated to fix go vet issues, passes tests now.

**Comment by jsha** [comment]
Ping on this?

**Comment by kisom** [comment]
LGTM.

**Pull request merged**